CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to node apps toolkit
+Please take a moment to review this document in order to make the contribution process easy and effective for everyone involved.
+
+## Getting started
+
+### Requirements
+
+- Node.js: `>=12.18.3`
+
+To install all dependencies and build all packages run the following commands from the root of the project.
+
+```
+npm
+npm run build
+```
+
+## Submitting a Pull Request
+
+Good pull requests, such as patches, improvements, and new features, are a fantastic help. They should remain focused in scope and avoid containing unrelated commits.
+
+Please ask first if somebody else is already working on this or the Contentful developers think your feature is in-scope. If one does not already exist then please create a related issue for the changes you plan to make.
+
+## Folder structure
+
+Source code and associated unit tests can be found in the `src/` directory. 
+Integration tests can be found in the `test/` directory.
+
+## Quality & Code Style
+
+### Commit messages
+
+All commit messages should meet the [conventional commit format](https://github.com/conventional-changelog/commitlint). 
+
+### Code formatting
+
+You don't need to worry about formatting your code. It is automatically reformatted using `prettier` on every commit using Git hooks.
+
+### Linting
+
+We use [ESLint](https://eslint.org/) and [Typescript ESLint](https://github.com/typescript-eslint/typescript-eslint) for linting and checking code for errors.
+
+All modern editors should automatically pick up configuration and show errors and warnings while you type.
+
+### Tests
+
+We use [Mocha](https://mochajs.org/) for writing tests.
+
+#### Setup tests
+Before running tests please run the `preTest` command. It will generate a keypair to enable testing of the related functionality.
```

Comment:
Shall we maybe point out that one shouldn't really worry about this? I am asking as we are probably removing this step as soon as we use the key generation endpoints in the `before*` blocks

```suggestion
Before running tests please run the `preTest` command. It will generate a keypair to enable testing of the related functionality. This will run automatically upon lunching `npm t`.
```